jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt

```diff
@@ -44,84 +33,12 @@ class CloudFormationStackNode(
         CloudFormationClient.SERVICE_NAME,
         stackName,
         AwsIcons.Resources.CLOUDFORMATION_STACK
-    ), ResourceParentNode {
+) {
     override fun resourceType() = "stack"
 
     override fun resourceArn() = stackId
 
     override fun displayName() = stackName
 
-    override fun isAlwaysLeaf(): Boolean = false
-
-    override fun isAlwaysShowPlus(): Boolean = true
```

Comment:
Should we set isAlwaysLeaf() to `true` since it is now always a leaf?

Reply:
The parent class has it as true